scripts/cargo_split.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Usage
+# ./scripts/tests.sh check|test "-F=feature1,feature2,..." <--no-run>
+
```

Comment:
This file should be named `tests.sh` right?

Reply:
It was named previously so, but now you can execute any cargo `<command>`. That's why I changed the name to another more generic.

Thinking now that maybe even already exists such cargo tool 🤔